stable/traefik/values.yaml

```diff
@@ -5,6 +5,14 @@ imageTag: 1.7.7
 serviceType: LoadBalancer
 # loadBalancerIP: ""
 # loadBalancerSourceRanges: []
+# default listener ports for load balancer
```

Comment:
I would prefer to have it under `service.ports.http`. Then all the service related pieces would be in one place:

```
service:
  ports:
    http: 80
    https: 443
  nodePorts:
    http: 30080
    https: 30443
```